smart-cards/src/main/java/gov/va/api/health/smartcards/patient/PatientController.java

```diff
@@ -228,4 +243,14 @@ private void lookupAndAttachLocations(Immunization.Bundle immunizations, String
               .collect(toList()));
     }
   }
+
+  @SneakyThrows
+  private String signVc(VerifiableCredential vc, String vcJws, String vcCompress) {
+    boolean shouldSign = parseBooleanOrTrue(vcJws);
+    boolean shouldDeflate = parseBooleanOrTrue(vcCompress);
```

Comment:
`deflate` vs `compress` is a little confusing to me. Can we standardize on `compress` and have `Deflater` just be an internal implementation detail of the `Compressors` class?